Carbon/util/LoginManager.js

```diff
@@ -0,0 +1,96 @@
+import AsyncStorage from '@react-native-async-storage/async-storage'
+
+
+
+function renderCallback() {
+    console.log("renderCallback not assigned in LoginManager.js, signin signout won't rerender.");
+};
+
+export async function getAuthHeader() {
+    return { headers: {"secret_token": await getToken() }};
+}
+
+export function setRenderCallback(cb) {
+    renderCallback = cb;
+}
+
+export async function getToken() {
+    return await AsyncStorage.getItem('secret_token');
+}
+
+export async function login(username, password) {
+    try {
+        // Working around url encode form because react native doesn't support it
+        var details = {
+            'email': username,
+            'password': password
+        };
+
+        var formBody = [];
+        for (var property in details) {
+            var encodedKey = encodeURIComponent(property);
+            var encodedValue = encodeURIComponent(details[property]);
+            formBody.push(encodedKey + "=" + encodedValue);
+        }
+        formBody = formBody.join("&");
+
+        var response = await fetch('http://192.168.0.150:3000/api/user/auth/login',
```

Comment:
Can this be changed to the config/api export? I think maybe for each link we just have them all declared there for specific routes.